src/main/java/mate/academy/bookstore/validation/FieldMatchValidator.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.bookstore.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import java.util.Objects;
+import mate.academy.bookstore.dto.UserRegistrationRequestDto;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch,
+        UserRegistrationRequestDto> {
+    @Override
+    public boolean isValid(UserRegistrationRequestDto requestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        return Objects.equals(requestDto.getPassword(), requestDto.getRepeatedPassword());

```

Comment:
This annotation works only with password and repeatedPassword, but it should work with 2 any fields, that is defined in annotation properties